app/main.py

```diff
@@ -42,6 +65,56 @@ async def index(request: Request):
     return RedirectResponse(f"{BASE_PATH}/docs")
 
 
+@app.get(f"{BASE_PATH}/check_for_dataset")
+@limiter.limit(RATE_LIMIT)
+async def check_for_dataset(request: Request):
+    """
+    Checks if a new dataset is available,
+    and downloads it
+    """
+    req = requests.get(
+        "https://api.github.com/repos/ambanum/OpenTermsArchive-versions/releases/latest"
```

Comment:
this could be in a global variable in case we change again the repo (which will happen)